### ISSUE TITLE
[SF][A11y][ui5-panel]: Visually apparent headings not made programmatic headings

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#5504]
### **Bug Description**
Visually apparent headings on Panel title not made programmatic headings

### **Expected Behavior**
Please use h tag instead of using role="heading" and aria-level

### **Steps to Reproduce**
1. Go to 'https://sap.github.io/ui5-webcomponents/playground/components/Panel/'
2. open JAWS
3. use JAWS shortcuts (INSERT+F6) for headings
4. you can't find Panel title "Both expandable and expanded" in JAWS heading list, it should be a h2
<img width="1065" alt="image" src="https://user-images.githubusercontent.com/13098801/178764839-e10def95-bbe3-4504-8504-5b7ba4dd7d6b.png">

### **Context**
- UI5 Web Components version: {1.4.0}
- OS/Platform: {...}
- Browser: {...}
- Affected component: {...}

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {SAP SuccessFactors}
- Business impact: {...}

**Comment by TeodorTaushanov** [comment]
Hello @SAP/ui5-webcomponents-topic-rl, take a look of this request.
